itou/templates/siae_evaluations/includes/list_item.html

```diff
@@ -46,16 +46,15 @@
             </div>
         </div>
 
-        {% if item.evaluated_siae.reviewed_at and item.state == 'REFUSED' or item.evaluated_siae.reviewed_at and item.state == 'REFUSED_2' %}
-            <div class="row mt-5">
-                <div class="col-md-12">
-                    <h3>Commentaires de la DDETS</h3>
-                    {{ item.labor_inspector_explanation }}
-
+        {% if not item.hide_state_from_siae %}
+            {% if item.evaluated_siae.reviewed_at and item.compute_state == 'REFUSED' or item.evaluated_siae.reviewed_at and item.compute_state == 'REFUSED_2' %}
+                <div class="row mt-5">
+                    <div class="col-md-12">
+                        <h3>Commentaires de la DDETS</h3>
+                        {{ item.labor_inspector_explanation }}
+                    </div>
```

Comment:
```suggestion
        {% if not item.hide_state_from_siae and item.evaluated_siae.reviewed_at %}
            {% with item_state=item.compute_state %}
                {% if item_state == 'REFUSED' or item_state == 'REFUSED_2' %}
                <div class="row mt-5">
                    <div class="col-md-12">
                        <h3>Commentaires de la DDETS</h3>
                        {{ item.labor_inspector_explanation }}
                    </div>
             ...
```
pour éviter un double `compute_state`